Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -106,8 +110,9 @@ class FakespotViewController: UIViewController, Themeable, UIAdaptivePresentatio
 
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
-        if presentingViewController == nil {
-            recordTelemetry()
+        if presentingViewController == nil,
+            viewModel.settingsCardViewModel.isReviewQualityCheckOn {
```

Comment:
This would mean we don't track when the user dismisses the bottom sheet when they didn't opt in yet but we should track that as well. The only thing that shouldn't be tracked is a dismissal due the user turning off the feature.

Reply:
Not sure I understand. If we turn off the feature by pressing "Turn Off Review Quality Check"(this will set the isReviewQualityCheckOn = false) we will still be able to see the shopping cart and open the bottom sheet?

Reply:
Yes, but if the user turns off the feature via settings and then opens the bottom sheet again and dismisses it we wouldn't track that dismissal.